zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4441,6 +4463,63 @@ components:
         email:
           description: The email of the user.
           type: "string"
+    CamundaUser:
+      type: "object"
+      properties:
+          userId:
+            description: The ID of the user.
+            type: "string"
+          userKey:
+            description: The system generated key of the User
+            type: "integer"
+            format: "int64"
+          displayName:
+            description: The display name of the User
+            type: "string"
+          authorizedApplications:
+            description: The applications the User is authorized to use
+            type: array
+            items:
+              type: "string"
+          tenants:
+            description: The tenants the User is a member of
+            type: array
+            items:
+              type: "object"
```

Comment:
Could this be a `TenantItem` ?

Reply:
I didn't opt for that object because I didn't want to expose the assigned members field also, just the id + name